Tasks/Xcode/task.json

```diff
@@ -12,7 +12,7 @@
     "version": {
         "Major": 2,
         "Minor": 0,
-        "Patch": 2
+        "Patch": 3
```

Comment:
Understood. Go for it :)

-----Original Message-----
From: "ericsciple" <notifications@github.com>
Sent: ‎11/‎1/‎2015 12:43 PM
To: "Microsoft/vso-agent-tasks" <vso-agent-tasks@noreply.github.com>
Cc: "scott dallamura" <scott.dallamura@gmail.com>
Subject: Re: [vso-agent-tasks] fix xcode workspace (#706)

In Tasks/Xcode/task.json:
> @@ -12,7 +12,7 @@
>      "version": {
>          "Major": 2,
>          "Minor": 0,
> -        "Patch": 2
> +        "Patch": 3
Let's change this to 4 instead. Trying out a patch for onedrive with patch version 3.
—
Reply to this email directly or view it on GitHub.